libs/wingsdk/src/std/array.ts

```diff
@@ -38,6 +38,19 @@ export class Array {
     throw new Error("Abstract");
   }
 
+  /**
+   * Try to get the value at the given index
+   *
+   * @macro ($self$[$args$])
```

Comment:
to handle negative indexes? idk I guess this stuff is out of scope for the focus of this PR.

```suggestion
   * @macro ($self$.at($args$))
```